pkg/query/adapter/adapter.go

```diff
@@ -0,0 +1,494 @@
+package adapter
```

Comment:
Please add the same license header as cmd/thanos/query.go

Reply:
done